install/apprise-api-install.sh

```
@@ -0,0 +1,61 @@
+color
+verb_ip6
+catch_errors
+setting_up_container
+network_check
+update_os
+
+PYTHON_VERSION="3.12" setup_uv
+fetch_and_deploy_gh_release "apprise" "caronc/apprise-api" "tarball"
+
+msg_info "Setup Apprise-API"
+cd /opt/apprise
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)